packages/n/node-vibrant.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "node-vibrant",
+  "filename": "vibrant.min.js",
+  "description": "Extract prominent colors from an image. Supports both node and browser environment.",
+  "keywords": [
+    "color",
+    "detection",
+    "varation",
+    "image",
+    "picture",
+    "canvas",
+    "vibrant",
+    "muted",
+    "colour"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/akfish/node-vibrant.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "node-vibrant",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": ["*.js"]
+      }
+    ]
+  },
+  "license": "MIT",
+  "author": "akfish"
```

Comment:
As the CI error notes, `author` isn't valid. You'll want an `authors` array with `name` inside each object.

Reply:
Thank you, I have updated it